contrib/python-bindings/include/cell_accessor_wrapper.h

```diff
@@ -89,6 +89,13 @@ namespace python
     PointWrapper
     get_barycenter() const;
 
+    /**
+     * Get the barycenter of the cell.
```

Comment:
```suggestion
     * Get the center of the cell.
```
You could also just copy the documentation from `TriaAccessor::center()` (via @copydoc).